docs/sources/alerting/create-alerts.md

```diff
@@ -9,7 +9,7 @@ weight = 200
 
 Grafana alerting allows you to attach rules to your dashboard panels. When you save the dashboard, Grafana extracts the alert rules into a separate alert rule storage and schedules them for evaluation.
 
-{{< imgbox max-width="1000px" img="/img/docs/alerting/drag_handles_gif.gif" caption="Alerting overview" >}}
+![Alerting overview](/img/docs/alerting/drag_handles_gif.gif)
```

Comment:
Is there a reason there is a `_gif` before the extension `.gif`?

Reply:
Because that's what it was named. I did not change the image names, just updated the link syntax.

There is a backlog item to update all the images in the section anyway, so the names can be updated when that happens.